packages/runner-ct/src/lib/webpack-hmr-client.ts

```diff
@@ -30,12 +36,14 @@ export function connectWebpackHmr ({ url, onReload }) {
 
       onReload()
     },
-    errors: (errors) => {
+    errors: (errors: string[]) => {
```

Comment:
🤦‍♂️ lets enable `noImplicitAny`

Reply:
We should be I think that'll need to be another PR - there are many things that need typing haha